code/modules/mining/mine_turfs.dm

```diff
@@ -303,6 +303,7 @@ turf/unsimulated/mineral/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_l
 						B = getFromPool(/obj/structure/boulder, src)
 						if(artifact_find)
 							B.artifact_find = artifact_find
+							B.investigation_log(I_ARTIFACT, "|| [B.artifact_find.artifact_find_type] || found by [user.name]([user.ckey]).")
```

Comment:
Yes, but `key_name(user)` is the correct way to do it, instead of manual formatting.